script.js

```diff
@@ -0,0 +1,60 @@
+//option 1
+/*
+var productsItems = products.items;
+var count = 0;
+productItems.forEach(function(item){
+  if (item.kind == "shopping#product")
+    count++;
+});
+
+console.log(count);
+
+
+//Q1 option 2
+var productsItems = products.items;
+function kindShoppingProduct(x) {
+  var kSP = 0;
```

Comment:
Try to name your variables more verbosely. So when others or when you read the code 3 months down the road, it'll be immediately understandable

Reply:
Noted! Thanks scott!